src/pybind/ceph_argparse.py

```diff
@@ -23,6 +23,9 @@
 import uuid
 
 
+public static final FLAG_MGR = 8;   # command is intended for mgr
```

Comment:
ha!  i googled 'python integer constant' and cut and pasted the stackoverflow questin instead of the answer :)